tests/unit/test-util.js

```diff
@@ -5,11 +5,18 @@ const camelCase = require('camelcase');
 const _ = require('lodash');
 const config = require('config');
 
-sinon.stub(config, 'get').returns({ oracledb: {} });
+const { fakeBaseUrl } = require('./test-data');
+
 const conn = appRoot.require('api/v1/db/oracledb/connection');
 const { openapi } = appRoot.require('utils/load-openapi');
-const { fakeBaseUrl } = require('./test-data');
 
+sinon.stub(config, 'get').returns({ oracledb: {} });
+/**
+ * @summary Create the stub for connection object that is useful for controlled testing environment.
+ * @function
+ * @param {Object} testCase The testcase that becomes the return value of connection.execute in the stub.
```

Comment:
We are transitioning to lowercase o on `object`. Can you make this change in your other classes?